kvmagent/kvmagent/plugins/virtiofs_plugin.py

```
@@ -205,7 +221,7 @@ def get_vm_domain(vm_uuid):
                 conn.close()
             except Exception:
                 pass
-        raise Exception("Failed to get VM domain[%s]: %s" % (vm_uuid, str(e))) from e
+        raise Exception("Failed to get VM domain[%s]: %s" % (vm_uuid, str(e)))
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)